### PR TITLE
Added close button on image stories

### PR DESCRIPTION
**Pull request by malavikakoppula**
## Description

when I expand an image full screen, I need a persistent close option to display on the screen, so that I can easily return to the article and have a consistent user experience when viewing images/galleries. 

## Jira Ticket

- [THEMES-893](https://arcpublishing.atlassian.net/browse/THEMES-893)

## Acceptance Criteria

GIVEN I am an Media Site User, WHEN I expand an image on an article, THEN:

- The 'X' close option persists in the left header of full-screen (consistent with linked designs)
- Styling is consistent with v2 designs on expanded image 

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-893`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/lead-art-block`
3. Click the fullscreen and to close it, click on close(X) button

## Dependencies or Side Effects
- Feature pack pr for local development: https://github.com/WPMedia/arc-themes-feature-pack/pull/375
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-893]: https://arcpublishing.atlassian.net/browse/THEMES-893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[THEMES-893]: https://arcpublishing.atlassian.net/browse/THEMES-893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[THEMES-893]: https://arcpublishing.atlassian.net/browse/THEMES-893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/lead-art-block/_index.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
 		@include scss.block-components("lead-art-carousel-image-wrapper");
 		@include scss.block-properties("lead-art-carousel-image-wrapper");
 	}
+	&__carousel-close-icon {
```
[the remark this answers is not in the document]
Comment (malavikakoppula) [reply blocks/lead-art-block/_index.scss]: Fixed

File: blocks/lead-art-block/features/leadart/default.jsx [hunk 2]
```diff
@@ -142,6 +142,11 @@ export const LeadArtPresentation = (props) => {
 							resizedOptions={{ auth: lead_art.auth[RESIZER_APP_VERSION], smart: true }}
 							resizerURL={RESIZER_URL}
 						/>
+						{isOpen ? (
+							<Button type="button" onClick={toggleFullScreen}>
```
Comment (vgalatro) [inline blocks/lead-art-block/features/leadart/default.jsx]: This should have an `accessibilityLabel` prop with the value `phrases.t("lead-art-block.fullscreen-exit")`.

Comment (vgalatro) [reply blocks/lead-art-block/features/leadart/default.jsx]: The button above on line `124` needs the prop to be fixed as well, from `label` to `accessibilityLabel`.

**Review by vgalatro — CHANGES_REQUESTED** [review]
This is looking pretty good, but noticed a couple of issues. First, there should be an accessibility label on the button. Second, it looks like the close button should be left aligned at desktop sizes in the mock ups, but I'm seeing it on the right:

Screen Shot 2023-01-03 at 1 16 18 PM

**Review by vgalatro — DISMISSED** [review]
Changes look good, approved!

File: blocks/lead-art-block/features/leadart/default.jsx (hunk 2 above)
Comment (malavikakoppula) [reply blocks/lead-art-block/features/leadart/default.jsx]: Did the changes

**Review by vgalatro — DISMISSED** [review]
Re-approving with the new changes.

File: .storybook/themes/news.scss [hunk 3]
```diff
@@ -2732,6 +2760,17 @@
 						),
 					),
 				),
+				lead-art-carousel-close-icon: (
+					right: calc(100% - 48px),
```
Comment (matthewroach) [inline .storybook/themes/news.scss]: Is `48px` the same as global-spacing-4 - so can this become - 

```suggestion
					right: calc(100% - var(--global-spacing-4)),
```

Comment (malavikakoppula) [reply .storybook/themes/news.scss]: @matthewroach Thanks, updated to global spacing

File: blocks/lead-art-block/_index.scss [hunk 4]
```diff
@@ -10,6 +10,11 @@
 		@include scss.block-components("lead-art-carousel-image-wrapper");
 		@include scss.block-properties("lead-art-carousel-image-wrapper");
 	}
+
+	&__carousel-close-icon {
+		@include scss.block-components("lead-art-carousel-close-icon");
+		@include scss.block-properties("lead-art-carousel-close-icon");
+	}
```
Comment (nschubach) [inline blocks/lead-art-block/_index.scss:17]: carousel is not relevant here...

File: blocks/lead-art-block/features/leadart/default.jsx [hunk 5]
```diff
@@ -139,9 +139,18 @@ export const LeadArtPresentation = (props) => {
 							width={800}
 							height={450}
 							responsiveImages={[800, 1600]}
-							resizedOptions={{ auth: lead_art.auth[RESIZER_APP_VERSION], smart: true }}
+							resizedOptions={{ auth: lead_art.auth[2], smart: true }}
 							resizerURL={RESIZER_URL}
```
Comment (nschubach) [inline blocks/lead-art-block/features/leadart/default.jsx]: If you pass `ansImage={lead_art}` to the `Image`, you don't need the auth token, url, or the resizer url and can remove this import (noted above)

You'd have to modify the other image too (`galleryItem`) to remove this dependency

Comment (nschubach) [reply blocks/lead-art-block/features/leadart/default.jsx]: I know this is not your doing, but `lead_art` should be `leadArt` to keep with our naming conventions.

Comment (nschubach) [reply blocks/lead-art-block/features/leadart/default.jsx]: Also, considering I recommend removing this so it doesn't matter, but we should never be hardcoding the `RESIZER_APP_VERSION`

**Review by nschubach — CHANGES_REQUESTED** [review]
We should, at least, not be hard coding the resizer version.

File: blocks/lead-art-block/features/leadart/default.jsx (hunk 5 above)
Comment (malavikakoppula) [reply blocks/lead-art-block/features/leadart/default.jsx]: @nschubach Thanks. Fixed this [bcfd60a](https://github.com/WPMedia/arc-themes-blocks/pull/1588/commits/bcfd60ad133d39765c9978ede17115524966fc72)

**Review by nschubach — APPROVED** [review]
Looks good!

**Review by vgalatro — APPROVED** [review]
Re-approving

**Pull request merged**